cmd/notary/keys.go

```diff
@@ -449,12 +430,11 @@ func (k *keyCommander) keysRotate(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	for _, role := range rolesToRotate {
-		if err := nRepo.RotateKey(role, k.rotateKeyServerManaged); err != nil {
-			return err
-		}
+	err = nRepo.RotateKey(rotateKeyRole, k.rotateKeyServerManaged)
+	if err == nil && k.rotateKeyServerManaged {
+		err = nRepo.Publish()
```

Comment:
One thought that we were discussing: is it possible for publish only the key rotation change, instead of all staged changes?  I'm not sure of the best way to go about doing this without making users published all staged changes prior to this one?